examples/HexagonalTopology.ipynb

```diff
@@ -1137,10 +494,157 @@
     "ax.legend(handles=legend_elements, bbox_to_anchor=(0.1, 1.08), loc='upper left', \n",
     "          borderaxespad=0., ncol=3, fontsize=14)\n",
     "\n",
-    "plt.savefig('resulting_images/som_seed_hex.png')\n",
+    "plt.savefig('resulting_images/som_seed.png')\n",
     "plt.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plotting: bokeh\n",
+    "Below, are plotting using bokeh to create an interactive hexagonal topology.\n",
```

Comment:
we are plotting

Reply:
Done! https://github.com/JustGlowing/minisom/pull/88/commits/750812ca6b4c30d53489b3f3381c63a10771475f